### PR TITLE
Bulk User Import Sample

### PR DESCRIPTION
**Pull request by JoneSabino**

### CODE REVIEW

File: bulk-user-import-sample/BulkUserImport/src/main/java/com/wso2/csa/BulkImportUsers.java [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.wso2.csa;
```
Comment (GDRDABARERA) [inline bulk-user-import-sample/BulkUserImport/src/main/java/com/wso2/csa/BulkImportUsers.java]: shall we add something like com.wso2.clients.bulkimport

File: bulk-user-import-sample/BulkUserImport/pom.xml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>8</source>
+                    <target>8</target>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+</project>
```
Comment (janakamarasena) [inline bulk-user-import-sample/BulkUserImport/pom.xml]: Let's add a new line at the end of the file.

File: bulk-user-import-sample/BulkUserImport/src/main/java/com/wso2/csa/BulkImportUsers.java [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+
+        out.println("Users import time: " +
+                minutes + "m " + seconds + "s " + millis + "ms");
+
+        out.println("Import process finished");
+
+        login.logOut();
+    }
+}
```
Comment (janakamarasena) [inline bulk-user-import-sample/BulkUserImport/src/main/java/com/wso2/csa/BulkImportUsers.java]: Let's add a new line at the end of the file.

File: bulk-user-import-sample/BulkUserImport/src/main/java/com/wso2/csa/ClientGetPropertiesValues.java [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    }
+
+    String getAuthUser() {
+        return authUser;
+    }
+
+    String getAuthPwd() {
+        return authPwd;
+    }
+}
```
Comment (janakamarasena) [inline bulk-user-import-sample/BulkUserImport/src/main/java/com/wso2/csa/ClientGetPropertiesValues.java]: Let's add a new line at the end of the file.

File: bulk-user-import-sample/BulkUserImport/src/main/java/com/wso2/csa/BulkImportUsers.java (hunk 1 above)
Comment (JoneSabino) [reply bulk-user-import-sample/BulkUserImport/src/main/java/com/wso2/csa/BulkImportUsers.java]: done

File: bulk-user-import-sample/BulkUserImport/pom.xml (hunk 2 above)
Comment (JoneSabino) [reply bulk-user-import-sample/BulkUserImport/pom.xml]: done

File: bulk-user-import-sample/BulkUserImport/src/main/java/com/wso2/csa/BulkImportUsers.java (hunk 3 above)
Comment (JoneSabino) [reply bulk-user-import-sample/BulkUserImport/src/main/java/com/wso2/csa/BulkImportUsers.java]: done

File: bulk-user-import-sample/BulkUserImport/src/main/java/com/wso2/csa/ClientGetPropertiesValues.java (hunk 4 above)
Comment (JoneSabino) [reply bulk-user-import-sample/BulkUserImport/src/main/java/com/wso2/csa/ClientGetPropertiesValues.java]: done

**Pull request merged**